auslib/admin/views/scheduled_changes.py

```diff
@@ -91,12 +92,14 @@ def _post(self, sc_id, form, transaction, changed_by):
             # from the rule (aka, set as NULL in the db). The underlying Form
             # will have already converted it to None, so we can treat it the
             # same as a modification here.
-            if (request.json and k in request.json):
+            if request.json and k in request.json:
                 what[k] = v.data
 
         where = {"sc_id": sc_id}
         try:
             self.sc_table.update(where, what, changed_by, form.sc_data_version.data, transaction)
+        except PermissionDeniedError as e:
+            return Response(status=403, response=str(e))
```

Comment:
so should we also stop catching valueError too ? : https://github.com/aksareen/balrog/blob/1f06024cfbcf96fe77febf56eeb1124c2ab58e0c/auslib/admin/views/scheduled_changes.py#L103